src/main/java/Tools.java

```diff
@@ -0,0 +1,29 @@
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Scanner;
+
+public class Tools {
+    public static void showCountryInfo(String countryId, Map<String, Country> countryMap) {
+        if (countryMap.get(countryId) == null) {
```

Comment:
Lepiej użyć metody `containsKey()` niż ręcznie porównywać do nulla, ewentualnie raz wywołaj metodę get() przed ifem, przypisz wynik do zmiennej i wykorzystaj to zarówno w ifie, jak i w dalszej części metody, aktualnie niepotrzebnie dwa razy wywołana jest metoda get